core-services/prow/02_config/openshift/kube-compare/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/kube-compare
+  require_self_approval: false
```

Comment:
Typically we set this "true" so that all PRs have to be explicitly approved rather than implicitly approved.

Reply:
changed it :)